docs/Usage.md

```diff
@@ -143,6 +143,12 @@ user@host ~/path/to/folder/py-kms $ python3 pykms_Server.py -V DEBUG
 user@host ~/path/to/folder/py-kms $ python3 pykms_Client.py -V DEBUG
 ```
 
+If you wish to get KMS server from DNS server: (ie perform a DNS resolution on _vlmcs._tcp.domain.tld, if ever there are several answers, only the first one is selected.)
```

Comment:
Could you extend, that this is only used IF Windows is in AD-mode? I think this would be appropriate to repeat here :grin:

Reply:
that's what I thought also, but if computer's full name and workgroup are set, there is no need to join an AD. activation is performed